public/index.html

```diff
@@ -13,6 +13,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="PolicyEngine" content="Compute the impact of public policy" />
+    <meta
+      name="description"
+      content="PolicyEngine&apos;s free, open-source software turns law into code."
```

Comment:
Thanks- could we change to `PolicyEngine's free, open-source software computes the impact of public policy`? @MaxGhenis WDYT